app/views/hosts/show.html.erb

```diff
@@ -46,9 +46,11 @@
           <%= render :partial => "hosts/metrics", :locals => { :report_summary => @report_summary[@host.name][:metrics] } %>
         <% end %>
       </div>
-      <div class="tab-pane" id="template" data-ajax-url='<%= templates_host_path(@host)%>'>
-        <%= spinner(_('Loading template information ...')) %>
-      </div>
+      <% if SETTINGS[:unattended] %>
```

Comment:
This means that the behaviour will change for `SETTINGS[:unattended]` - instead of a message this div will not be displayed at all.